src/SlackMessages.ts

```diff
@@ -253,3 +246,97 @@ export function rugButtonFrom(action: ButtonSpec, command: IdentifiableInstructi
     }
     return button;
 }
+
+interface StandardAttachmentDefaults {
```

Comment:
I'm happy to move it to a different file. I just though because these were Slack messages, they belonged in SlackMessages.